crates/katana/core/src/backend/config.rs

```diff
@@ -1,6 +1,6 @@
 use std::path::PathBuf;
 
-use ::primitive_types::U256;
```

Comment:
we can remove `primitive_types` from `Cargo.toml` now

Reply:
Done